resilience4j-retry/src/main/java/io/github/resilience4j/retry/Retry.java

```diff
@@ -510,9 +510,16 @@ default void executeRunnable(Runnable runnable) {
 
 		/**
 		 * Records a successful call.
+		 * @deprecated since 1.2.0
 		 */
+		@Deprecated
 		void onSuccess();
 
+		/**
+		 * Records a successful call or retryable call with the needed generated retry events.
+		 */
+		void onFinish();
```

Comment:
One last request. Can we rename it to `onComplete`? :)

Reply:
Sure i suck in naming :)